### PR TITLE
DAOS-9864 test: Remove setnames variants from pool/simple_create_dele…

### PR DESCRIPTION
**Pull request by shimizukko**
…te.py

Remove setnames variants from pool/simple_create_delete.py because it’s not used.

Skip-unit-tests: true
Test-tag: simple_create
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

File: src/tests/ftest/pool/simple_create_delete.py [hunk 1]
```diff
@@ -23,8 +23,9 @@ def test_create(self):
         Test basic pool creation.
 
         :avocado: tags=all,full_regression
-        :avocado: tags=tiny
-        :avocado: tags=pool,smoke,simple_create
+        :avocado: tags=vm
+        :avocado: tags=pool
+        :avocado: tags=smoke,simple_create
```
Comment (daltonbohning) [inline src/tests/ftest/pool/simple_create_delete.py:28]: Minor/optional - I think `smoke` makes more sense with "features":
```
        :avocado: tags=all,full_regression
        :avocado: tags=vm
        :avocado: tags=pool,smoke
        :avocado: tags=simple_create
```

**Review by daltonbohning — APPROVED** [review]
LGTM

**Review by dinghwah — APPROVED** [review]
LGTM
Please update the description header.

**Review by daltonbohning — APPROVED** [review]

File: src/tests/ftest/pool/simple_create_delete.py (hunk 1 above)
Comment (shimizukko) [reply src/tests/ftest/pool/simple_create_delete.py]: Done

**Pull request merged**